src/main/java/Duke.java

```diff
@@ -25,7 +25,7 @@ public static void main(String[] args) {
 
             Object[] parseResult = InputParser.parseInput(input);
             InputType inputType = (InputType) parseResult[0];
-            String value = (String) parseResult[1];
+            String[] value = (String[]) parseResult[1];
```

Comment:
Why the cast to String[] array here, instead of making `InputParser::parseInput(input)` return a String[] instead of Object[]?